store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/retriever/impl/AccumuloIDWithinSetRetrieverTest.java

```diff
@@ -322,16 +387,15 @@ private void shouldDealWithFalsePositives(final boolean loadIntoMemory, final Ac
         // Test random items against it - should only have to shouldRetrieveElementsInRangeBetweenSeeds MAX_SIZE_BLOOM_FILTER / 2 on average before find a
         // false positive (but impose an arbitrary limit to avoid an infinite loop if there's a problem).
         int count = 0;
-        int maxNumberOfTries = 50 * store.getProperties().getMaxBloomFilterToPassToAnIterator();
+        final int maxNumberOfTries = 50 * store.getProperties().getMaxBloomFilterToPassToAnIterator();
         while (count < maxNumberOfTries) {
             count++;
             if (filter.membershipTest(new Key(("" + count).getBytes()))) {
                 break;
             }
         }
-        if (count == maxNumberOfTries) {
-            fail("Didn't find a false positive");
-        }
+
+        assertThat(count).isNotEqualTo(maxNumberOfTries);
```

Comment:
Really minor thing (and I should have mentioned this before), if you want to keep the behaviour the exact same as before so it has the same fail message you could even do:
```java
assertThat(count)
        .isNotEqualTo(maxNumberOfTries)
        .withFailMessage("Didn't find a false positive");
```

Reply:
Added the failure message